controllers/categoryController.go

```diff
@@ -82,9 +84,16 @@ func GetCategories(categoryRepository *data.CategoryRepository) func(http.Respon
 			)
 			return
 		}
-		w.Header().Set("Content-Type", "application/json")
-		w.WriteHeader(http.StatusOK)
-		w.Write(j)
+		ctx := NewContext(context.Background(), j)
```

Comment:
Looks like we should try `r.Context()` instead of `context.Backgroun()`
[https://github.com/urfave/negroni/issues/90](https://github.com/urfave/negroni/issues/90)